tasks/go.py

```diff
@@ -650,3 +651,72 @@ def create_module(ctx, path: str, no_verify: bool = False):
         print(color_message("Not removing changed files", "red"))
 
         raise Exit(code=1) from e
+
+
+@task(iterable=['targets'])
+def mod_diffs(_, targets):
+    """
+    Lists differences in versions of libraries in the repo,
+    optionally compared to a list of target go.mod files.
+
+    Parameters:
+    - targets (str): Comma-separated list of paths to target go.mod files.
+    """
+    # Find all go.mod files in the repo
+    all_go_mod_files = []
+    for module in DEFAULT_MODULES:
+        all_go_mod_files.append(os.path.join(module, 'go.mod'))
+
+    # Validate the provided targets
+    for target in targets:
+        if target not in all_go_mod_files:
+            print(f"Error: Target go.mod file '{target}' not found.")
+            return
```

Comment:
Could you print this on stderr / raise an Exception ?

Reply:
yup